frontend-react/src/components/MessageTracker/QualityFilters.tsx

```diff
@@ -13,44 +12,21 @@ export const QualityFilters = ({
     return (
         <>
             <section className="margin-bottom-5">
-                <h3>Jurisdictions:</h3>
-                {qualityFilters?.map((d) => (
-                    <React.Fragment key={d.organization_id}>
-                        <table
-                            className="usa-table usa-table--borderless usa-table--striped width-full"
-                            data-testid="qualityFilter-table"
-                        >
-                            <thead>
-                                <tr className="text-baseline">
-                                    <th>
-                                        {d.organization} <br />
-                                        <span className="font-sans-3xs text-normal">
-                                            {" "}
-                                            ({d.filteredReportItems.length})
-                                            record(s) filtered out
-                                        </span>
-                                    </th>
-                                    <th>Field</th>
-                                    <th>Tracking element</th>
+                <h3>Quality Filters:</h3>
```

Comment:
This could get confusing for screen readers. The "Receivers" title is an h3, too, but this will be present as subsections under that title